### PR TITLE
🔀 chore(package.json): update date-fns dependency to version ^2.30.0 

### PR DESCRIPTION
**Pull request by ejcenteno**
Update date-fns dependency to version ^2.30.0 for compatibility and potential bug fixes

The date-fns dependency has been updated to version ^2.30.0 to ensure compatibility with the latest features and bug fixes.

### CODE REVIEW

File: DOC.md [hunk 1]
```diff
@@ -230,25 +230,25 @@ Format any valid date using date-fns' `format()` function.
...
 
 ```javascript
 const date = new Date('1976-03-20');
 
-const formatted = this.$morphDateFormat(date, 'MMM DD, YYYY');
-const year = this.$morphDateFormat(date, 'YYYY');
-const time = this.$morphDateFormat(new Date(), 'hh:mm A');
+const formatted = this.$morphDateFormat(date, 'MMM dd, yyyy');
+const year = this.$morphDateFormat(date, 'yyyy');
+const time = this.$morphDateFormat(new Date(), 'hh:mm a..aa');
```
Comment (jofftiquez) [inline DOC.md]: Is this really the correct format?

**Review by jofftiquez — CHANGES_REQUESTED** [review]
Please see comments. Thanks.

File: DOC.md (hunk 1 above)
Comment (ejcenteno) [reply DOC.md]: Ow sorry it shout be `a` only.

**Comment by ejcenteno** [comment]
Here's the result of the unit testing,
image

**Review by jofftiquez — APPROVED** [review]

**Comment by jofftiquez** [comment]
@ejcenteno publish in [v1.2.1](https://www.npmjs.com/package/vue-morphling/v/1.2.1) thanks!

**Pull request merged**